### ISSUE TITLE
Support for 'default' properties?

### ISSUE THREAD

**Issue by DeanThomas1983** [qmlnet/qmlnet#194]
Hi, I've been watching this project for quite a while and it looks like a great solution to cross platform GUIs for C#.  One thing I'm particulary interested in is being able to create a generic QML documents for use with C++, Python (PySide2), and C# models.

One thing I've not been able to work out so far with qml.net is if there is support for defining a 'default' property in a C# model.  In C++ this would be achieved using:
`Q_CLASSINFO("DefaultProperty", "content")`

Is there any way to do this currently in qml.net?  I'd guess you could use an attribute to define a default property, but wasn't able to see anything within the sources.  This seems like a similar open issue: https://github.com/qmlnet/qmlnet/issues/84

If it isn't currently possible, I'd be interested in giving it a go.

**Comment by pauldotknopf** [comment]
That isn't currently supported.

How would it look? Someting like this?

```csharp
[ClassInfo("Prop", "Value")]
public class NetClass
{
}
```

**Comment by DeanThomas1983** [comment]
Yeah, that was the way I was thinking it could work.  I'll look into pulling the code to see if I can work out how it would be implemented.
